scripts/sonar/sonar-controller.js

```diff
@@ -41,23 +50,51 @@ angular.module('app.sonar').controller('SonarCtrl', ['$scope', '$log', '$q', 'ap
                 var totalLines = sum(timePeriod.metrics, 'lines');
 
                 var weightedMetrics = timePeriod.metrics.map(function(project) {
-                    var factor = project.lines / totalLines;
 
                     return {
-                        methodComplexity: factor * project.methodComplexity,
-                        fileComplexity: factor * project.fileComplexity,
-                        coverage: factor * project.coverage,
-                        compliance: factor * project.compliance,
+                        numberOfMethodsInProjectInPeriod: project.totalMethods,
```

Comment:
I'm all for descriptive variable names, but these ones are probably a bit too wordy. Without 'InProjectInPeriod' I think they'd be fine, since that's implied from the context of the rest of the code that these are per project per time period.